---
title: "Final Report"
author: "Paul Brunner, Marie Kleinert, Felipe Stünkel, Chloé Weiler"
date: "16/06/2022"
output: html_document
bibliography: [packages.bib, Citations.bib]
---

# Abstract

# Introduction

# Methods

## Overview of used packages

+--------------+--------------------------------------------------------------------------------------------+-----------------+
| Package Name | Application                                                                                | **Reference**   |
+==============+============================================================================================+=================+
| babyplots    | create interactive 3D visualizations                                                       |                 |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| base         | basic R functions                                                                          | @R-base         |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| bayesbio     | calculate Jaccard coefficients                                                             | @bayesbio       |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| biomaRt      | access to genome databases                                                                 | @biomaRt2009    |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| cinaR        | combination of different packages                                                          | @cinaR          |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| cluster      | cluster analysis of data                                                                   | @cluster        |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| edgeR        | assess differential expression in gene expression profiles                                 | @edgeR          |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| enrichplot   | visualization of gene set enrichment results (GSEA)                                        | @enrichplot     |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| FactoMineR   | perform principal component analysis (PCA)                                                 | @FactoMineR     |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| fgsea        | Run GSEA on a pre-ranked list                                                              | @fgsea          |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| ggplot2      | visualization of results in dot plots, bar plots and box plots                             | @ggplot2        |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| ggpubr       | formatting of ggplot2-based graphs                                                         | @ggpubr         |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| grid         | implements the primitive graphical functions that underlie the ggplot2 plotting system     | @grid           |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| gridExtra    | arrange multiple plots on a page                                                           | @gridExtra      |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| gtools       | calculate foldchange, find NAs,                                                            | @R-gtools       |
|              |                                                                                            |                 |
|              | logratio2foldchange                                                                        |                 |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| knitr        | creation of citations using write_bib                                                      | @knitr2014      |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| limma        | "linear models for microarray data"                                                        | @limma2015      |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| msigdbr      | provides the 'Molecular Signatures Database' (MSigDB) gene sets                            | @msigdbr        |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| parallel     | allows for parallel computation through multi core processing                              | @parallel       |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| pheatmap     | draw clustered heatmaps                                                                    | @pheatmap       |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| RColorBrewer | provides color schemes for maps                                                            | @RColorBrewer   |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| scales       | helps in visualization: r automatically determines breaks and labels for axes and legends  | @scales         |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| Seurat       | visualize gene set enrichment results in dot plots                                         | @R-SeuratObject |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| tidyverse    | collection of R packages, including ggplot2                                                | @tidyverse      |
+--------------+--------------------------------------------------------------------------------------------+-----------------+
| uwot         | performs dimensionality reduction and Uniform Manifold Approxiamtion and Projection (UMAP) | @uwot           |
+--------------+--------------------------------------------------------------------------------------------+-----------------+

: **Tab. 1: Used packages in alphabetical order.**

## Principal Component Analysis (PCA)

## Uniform Manifold Approximation and Projection (UMAP)

## Gene Set Enrichment Analysis (GSEA)

## Gene Set Variation Analysis (GSVA)

# Results

# Conclusion
