---
title: " "
output:
  pdf_document:
    number_sections: TRUE
  html_document:
    code_folding: hide
documentclass: scrreprt
classoption: [parskip, oneside] # für drucken twoside
header-includes:
  - \usepackage[ngerman, main=english]{babel}
  - \usepackage[utf8]{inputenc}
  - \usepackage[T1]{fontenc}
  - \usepackage{lmodern}
  - \usepackage[onehalfspacing]{setspace}
  - \usepackage[left=2.50cm, right=2.50cm, top=2.50cm, bottom=2.50cm, bindingoffset=10mm, includehead, includefoot]{geometry}
  - \usepackage[headsepline]{scrlayer-scrpage}
  - \usepackage{url}
  - \usepackage[backend=biber, style=authoryear, giveninits=true, maxbibnames=99, uniquename=init, maxcitenames=2, hyperref=true, date=year]{biblatex}
  #- \usepackage{bib/cellstyle_it}
  - \usepackage{setspace}
  - \usepackage{xpatch}
  - \usepackage{csquotes}
  - \usepackage{amsmath}
  - \usepackage{listings}
  - \usepackage{booktabs}
  - \usepackage{longtable}
  - \usepackage{multirow}
  - \usepackage{rotating}
  - \usepackage{subfigure}
  - \usepackage{graphicx}
  - \usepackage{float}
  - \usepackage{acronym}
  - \usepackage{lipsum}
  - \usepackage{scrhack}
  - \emergencystretch=50pt
  - \clubpenalty = 10000
  - \widowpenalty = 10000
  - \displaywidowpenalty = 10000
  - \automark[section]{chapter}
  - \renewcommand*{\chaptermarkformat}{}
  - \renewcommand*{\sectionmarkformat}{}
  - \setkomafont{title}{\sffamily}
  - \setkomafont{disposition}{\usekomafont{title}}
  - \setkomafont{author}{\usekomafont{title}}
  - \setkomafont{date}{\usekomafont{title}}
  - \setkomafont{caption}{\sffamily\small}
  - \setkomafont{captionlabel}{\usekomafont{caption}\bfseries\small}
  - \setkomafont{pagehead}{\normalfont\scshape}
csl: [report/cell.csl]
bibliography: [report/packages.bib, report/Citations.bib]
---

```{=tex}
\begin{titlepage}
\centering
    {\Large Ruprecht-Karls-Universität Heidelberg\\
        Fakultät für Biowissenschaften\\
        Bachelorstudiengang Molekulare Biotechnologie\\}

    {\vspace{\stretch{2}}}
    {\doublespacing{\usekomafont{title}}

        {\Huge Cancer Hallmark and Metabolic Pathways in Cancer}

      {\Huge Topic 02 Team 03}

        {\Huge Exploration of Lung Adenocarcinoma (LUAD)}

    }

    \vspace{\stretch{2}}
    {\Large Data Science Project SoSe 2022}

    \vspace{\stretch{2}}

    {\Large
        \begin{tabular}{rl}
            Autoren & Paul Brunner, Marie Kleinert, Felipe Stünkel, Chloé Weiler\\
            Abgabetermin &18.07.2022\\
        \end{tabular}
    }

    \vspace{\stretch{1}}

\end{titlepage}
```
# Introduction

## Biological Background

## The Pan Cancer Project

The Cancer Genome Atlas (TCGA) is a publicly available collection of datasets that store the most important cancer-causing genomic alterations in order to create an 'atlas' of cancer genomic profiles. [@tomczak2015]. In 2012 TCGA Research Network launched the Pan-Cancer analysis project as a globally coordinated initiative whose main objective is to assemble coherent, consistent TCGA data sets across twelve different tumor types, one of which being lung adenocarcinoma (LUAD). Each tumor type is characterized using six different genomic, proteomic, epigenomic, and transcriptional platforms. Data collected from thousands of patients is analysed and interpreted in an attempt to gain a deeper understanding of the genomic changes that drive a normal cell to become cancerous. In the future, the aim is to analyse additional tumor types beyond the twelve original ones in the hopes that the pan cancer project will one day inform clinical decision-making and aid in the development of novel therapeutic options \[@weinstein2013\].

## Jaccard index

The Jaccard index is a widely known measurement for the similarity between finite sample sets. The restricted domain is defined between zero and one. A Jaccard index close to one indicates a high similarity of the sample sets \[@jaccard1901\].

$$
J(A,B)
= \frac{A \cap B}{A \cup B}
$$

## Principal Component Analysis (PCA)

Principal component analysis (PCA) is a procedure used to perform linear dimension reduction. The goal is to reduce the dimension of a given data set whilst losing as little information as possible by retaining a maximum of the standardized data set's variation [@ringner2008].

Principal components (PC) are a set of new orthogonal variables that are made up of a linear combination of the original variables. Principal components display the pattern of similarity of the observations and of the variables as points in maps [@abdi2010]. By convention, the PCs are ordered in decreasing  order according to the amount of variation they explain of the original data [@ringner2008]. It is important to note that all PCs are uncorrelated.

PCA is a useful tool for genome-wide expression studies and often serves as a first step before clustering or classification of the data. Dimension reduction is a necessary step for easy data exploration and visualization [@ringner2008].

## Uniform Manifold Approximation and Projection (UMAP)

Uniform manifold approximation and projection (UMAP) is a k-neighbour graph based algorithm that is used for nonlinear dimension reduction [@smets2019] [@mcinnes2018].

After data normalization, the Euclidean distances between points in a two-dimensional space of the graph are calculated and a local radius is determined [@VERMEULEN2021119547]. In general the closer two points are to each other, the more similar they are. UMAP makes a density estimation to find the right local radius. This variable radius is smaller in high density regions of data points and larger in low density regions. In general, the density is higher when the k-nearest neighbour is close and vice versa. The number of k-nearest neighbours controls the number of neighbours whose local topology is preserved. Precisely, a large number of neighbours will ensure that more global structure is preserved whereas a smaller number of neighbours will ensure the preservation of more local structure [@mcinnes2018].

UMAP is a newer method tan PCA and it is generally believed to be easier to interpret and group data than by using PCA. Furthmore, UMAP has the advantage of not requiring linear data [@milosevic2022].

## Gene Set Enrichment Analysis (GSEA)

Gene set enrichment analysis (GSEA) is a computational method that is used to determine whether two gene expression states are significantly different from each other or not. In this project we compared gene expression profiles between healthy and tumorous tissue of LUAD[@subramanian2007].

Two data sets are compared and the genes are sorted from the most to the least differential expression between the data sets according to their p-values. This creates a ranked list L.

Referring to an *a priori* defined set of gene sets S, the goal is to locate for each pathway of S where its corresponding genes fall in L and find a discerning trend. If the genes of a given pathway are randomly distributed in L then the pathway is assumed to not significantly contribute to the particular tumor's phenotype. However if the genes are primarily clustered at the top or the bottom of L then a phenotypic significance of the given pathway can be assumed.

To determine the location of the genes, an enrichment score is calculated for each pathway. For this, a running-sum statistic is calculated as the list L is ran through. The running-sum is increased every time a gene belonging to the pathway in question is encountered and decreased otherwise. An enrichment score is thus calculated for each pathway. The enrichment score is defined as the maximum deviation from zero of the running-sum.

Lastly, adjustment for multiple hypothesis testing is performed by normalizing the enrichment score for each pathway to account for its size and a normalized enrichment score is obtained.

GSEA is a useful tool for interpretation of gene expression data.

[@subramanian2005]

## Gene Set Variation Analysis (GSVA)

Gene set variation analysis (GSVA) is an unsupervised method to estimate pathway activities based on gene expression data. Contrarily to the aforementioned GSEA, GSVA does not rely on phenotypic characterisation of the data sets into two categories but rather quantifies enrichment in a sample-wise manner which makes GSVA the better choice to perform on the tcga_exp data set.

GSVA estimates a cumulative distribution for each gene over all samples. The gene expression values are then converted according to these estimated cumulative distributions into scaled values. Based on these new values, the genes are ranked in each sample. Next, the genes are classified into two distributions and a Komogorow-Smirnow statistic is calculated to judge how similar the distributions are to each other and to obtain an ES.

The GSVA corresponds to either the maximum deviation between both running sums or the GSVA score can be defined as the difference of the maximum deviations in the positive and in the negative direction. A highly positive or negative GSVA score indicates that the studied gene set is positively or negatively enriched compared to the genes not in the gene set, respectively. If the GSVA score for a given gene set is close to zero, then the gene set is probably not differentially expressed compared to the genes not in the gene set.

[@hanzelmann2013]

# Abbreviations

|         |                                               |
|---------|-----------------------------------------------|
| GSEA    | gene set enrichment analysis                  |
| GSVA    | gene set variation analysis                   |
| LUAD    | lung adenocarcinoma                           |
| PC      | principal component                           |
| PCA     | principal component analysis                  |
| RNA-seq | RNA sequencing                                |
| TCGA    | The cancer genome atlas                       |
| UMAP    | uniform manifold approximation and projection |

# Methods

## Our Data

At the beginning of this project we were given four datasets of which two contained RNA-seq data, one contained clinical annotations pertaining to one of the RNA-seq data frames and one contained a list of gene sets for cancer hallmark analysis.

The first RNA-seq dataset is a data frame containing RNA-seq data from almost 10,000 TCGA cancer patients for 33 different tumor types. The data stored within that data frame was used to perform pan cancer analysis and to create a logistic regression model. The second RNA-seq dataset is a smaller data frame containing the TCGA expression data of tumor tissue and the corresponding healthy tissue for five different cancer types. A focused analysis was performed on this second dataset.

RNA sequencing (RNA-seq) data makes it possible to go beyond static genome analysis and to gain an insight into the transcriptional landscape of a cell. Studying gene expression profiles of a given cell, regulated by RNA synthesis, enables researchers to gain a deeper understanding of how gene expression is regulated in cells and its impact on the cell's phenotype [@marguerat2010].

All expression data was log2(TPM) transformed. Log2 Transformation is a commonly used tool to reduce skewness in data and to make it more conform to a normal distribution. Here, TPM stands for 'Transcripts per million' and refers to a method of RNA-seq normalization in which one first accounts for gene length before adjusting for sequencing depth, or count bias, in the data. A possible perk of TPM is the reduction of type I and type II errors which would otherwise falsify downstream analysis results by accounting for gene length first [@yuen2022].

## Overview of used packages

+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| Package Name    | Application                                                                                | **Reference**      |
+=================+============================================================================================+====================+
| babyplots       | create interactive 3D visualizations                                                       | @babyplots         |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| base            | basic R functions                                                                          | @R-base            |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| bayesbio        | calculate Jaccard coefficients                                                             | @bayesbio          |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| BiocParallel    | novel implementations of functions for parallel evaluation                                 | [@biocparallel]    |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| biomaRt         | access to genome databases                                                                 | @biomaRt2009       |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| blorr           | building and validating binary logistic regression models                                  | @blorr             |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| cinaR           | combination of different packages                                                          | @cinaR             |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| cluster         | cluster analysis of data                                                                   | @cluster           |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| ComplexHeatmap  | arrange multiple heatmaps                                                                  | [@complexheatmap]  |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| edgeR           | assess differential expression in gene expression profiles                                 | @edgeR             |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| EnhancedVolcano | produce improved volcano plots                                                             | [@enhancedvolcano] |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| enrichplot      | visualization of gene set enrichment results (GSEA)                                        | @enrichplot        |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| FactoMineR      | perform principal component analysis (PCA)                                                 | @FactoMineR        |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| fgsea           | Run GSEA on a pre-ranked list                                                              | @fgsea             |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| ggplot2         | visualization of results in dot plots, bar plots and box plots                             | @ggplot2           |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| ggpubr          | formatting of ggplot2-based graphs                                                         | @ggpubr            |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| grid            | implements the primitive graphical functions that underlie the ggplot2 plotting system     | @grid              |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| gridExtra       | arrange multiple plots on a page                                                           | @gridExtra         |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| GSVA            | Run GSVA on a data set                                                                     | [@gsva]            |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| gplots          | plotting data                                                                              | [@gplots]          |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| gtools          | calculate foldchange, find NAs,                                                            | @R-gtools          |
|                 |                                                                                            |                    |
|                 | logratio2foldchange                                                                        |                    |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| knitr           | creation of citations using write_bib                                                      | @knitr2014         |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| limma           | "linear models for microarray data"                                                        | @limma2015         |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| msigdbr         | provides the 'Molecular Signatures Database' (MSigDB) gene sets                            | @msigdbr           |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| parallel        | allows for parallel computation through multi core processing                              | @parallel          |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| pheatmap        | draw clustered heatmaps                                                                    | @pheatmap          |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| RColorBrewer    | provides color schemes for maps                                                            | @RColorBrewer      |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| ROCR            | visualizing classifier performance                                                         | @rocr              |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| scales          | helps in visualization: r automatically determines breaks and labels for axes and legends  | @scales            |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| Seurat          | visualize gene set enrichment results in dot plots                                         | @R-SeuratObject    |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| tidyverse       | collection of R packages, including ggplot2                                                | @tidyverse         |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| uwot            | performs dimensionality reduction and Uniform Manifold Approxiamtion and Projection (UMAP) | @uwot              |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+

: **Tab. 1: Used packages in alphabetical order.**

## Gene Set Extraction

The Molecular Signature Database (MSigDB) is a database offering a variety of annotated gene sets publicly available for analysis. The import of gene sets from MSigDB into RStudio can easily be performed using the R package "msigdbr" [@msigdbr] which allows the extraction of species-specific gene sets of the category of interest. In a final step, the prefix corresponding to the source of the gene set was removed. The resulting output was a list containing all selected gene sets with the comprising genes saved in a vector and each element named after the pathway stored in it. 
This workflow was conducted to extract curated (C2) and ontology (C5 BP) gene sets which were used for focused analysis as well as pan cancer analysis. The curated gene sets that regulate the metabolism of cells were also used for comparison with known pathways deregulated in cancer cells.

## Data cleanup on TCGA expression dataset

In order to enable an efficient workflow on the big TCGA dataset containing cancer patient RNA-seq expression data, the total amount of genes had to be reduced. In a first cleanup step the amount and distribution of NA's were analysed. There were no NA's in the dataset itself, which means that no patients had to be removed. Next all genes that showed a very low standard deviation were removed from the dataset. As a cutoff value we used the value of the 50% quantile of the standard deviation distribution.

To decrease the amount of genes even further and focus our analysis on important genes a biotype analysis was conducted. Using the BiomaRt package [@biomaRt2005] the biotypes of all genes in the dataset were retrieved and compared to biotypes of the given geneset list, as well as the geneset lists we retrieved from MSigDB using the native msigdbr package [@msigdbr]. All genes linked to biotypes that were not found in the biotypes of said genesets and possible pseudogenes were removed. However genes belonging to lncRNA or siRNA were kept as they also have a significance in various biological processes. \[@ondrej2016\].

In the end the expression dataset could be reduced from over 60.000 genes to roughly 17.000, while keeping all 9741 patients.

## Pancancer analysis

### Dimensionality reduction

The first step to pan cancer comparison was to evaluate potential clusters in our data. To uncover these, a combination of a PCA and UMAP was conducted on the cleaned tcga expression dataset. First the PCA using the RunPCA command from the Seurat [@R-SeuratObject] package and later on the UMAP analysis was done on the produced principle components using the uwot [@uwot] package. Performing the PCA before the UMAP analysis was necessary to minimize artefacts caused by correlating variables in our dataset. Principle components do not correlate by nature. The UMAP results per patient were then plotted with ggplot2 [@ggplot2] and colored by their corresponding tumor type to gain insight into cluster formation.

The dataset was subsetted into dataframes containing only patients of one cancer type. The aforementioned workflow was then used on each of the cancer type subsets. All of the created plots were then collected into one overview of intra-cancer clusters. To analyse these clusters k-means \[\@R-base\] clustering was performed to assign each of the patients their corresponding cluster. The ideal number of clusters was evaluated using the silhouette method using the function from the cluster package \[\@cluster\]. Based on these assignments foldchanges between each cluster and the rest of the patients were calculated using the foldchange function [@gtools]. Additionally, a two sided Wilcoxon test was conducted on each cluster and the rest. These two metrics were used to create a volcano plot showing the -log10 of the Wilcoxon p-values plotted against log2 foldchanges of each gene between the clusters. These plots were created using the EnhancedVolcano package [@enhancedvolcano]. From these plots the most significant and over or under expressed genes were extracted.

### GSVA

In order to compare cancer types with each other, the genes needed to be condensed in more easily understandable metrices. For this we evaluated the enrichment of genes that play a role in the same pathways over all cancer types by conducting a GSVA. Two different geneset lists were used: gene ontology genesets and curated genesets, both retrieved from msigdb, as mentioned before. Genesets that showed a low overlap with the dataset genes, meaning below 95% of the pathways genes could be found in the dataset, were removed. Going into the GSVA, both geneset lists consisted of roughly 3000 genesets. To conduct the GSVA a package of the same name was used, that also enables multi-core calculations. This was crucial in order to cut down on calculation time [@gsva]. Afterwards, pathways that showed a low standard deviation were removed from the resulting pathway enrichment matrix. The patient pathway enrichment matrix was turned into a tumor type pathway enrichment matrix by subsetting it into the different tumor types and calculating the means per pathway over all patients of one tumor type. A heatmap was produced using the ComplexHeatmap package [@complexheatmap]. To compare these two geneset lists concerning their information value, this workflow was repeated for each of them.

For quality control, a PCA and UMAP were used on the newly created pathway patient enrichment matrices, to check whether the clusters that could be seen on the first UMAP plot could partly be found again. After quality control we decided to keep working with the gene ontology genesets (C5 BP).

After subsetting the pathway patient enrichment matrix into one data frame for LUAD and the other cancer types, foldchanges were calculated and a two sided Wilcoxon test was conducted between the means per pathway for all LUAD patients and all other patients. With these values another volcano plot was created and the interesting pathways were extracted.

The resulting heatmap and volcano plot could then be used to gather information of the different molecular signatures of LUAD and other cancer types and gave critical information for the following regression analysis.

## Regression

To use the information gathered by the pan cancer analysis a regression model with the purpose of identifying potential LUAD patients from RNA-Seq data was added to the project. In order to make the binary decision between LUAD and non-LUAD patients a logistic regression model was trained. First of all the cleaned dataset was split 70/30 into a training and testing dataset. Additionally every LUAD patient in these datasets was marked with a 1 and non-LUAD patients with a 0, so that the model can be evaluated later on.

In an effort to find genes that could be used as explaining variables, gene foldchanges \[@gtools\] between LUAD and other cancer types were calculated using the cleaned dataset. The genes were additionally tested for correlation and for all highly correlating genes, one of them was removed from the dataset. The 10 most overexpressed and 10 most underexpressed genes were chosen for further testing. As a quality control PCA [@R-SeuratObject] and UMAP [@uwot] were conducted on all patients for these 20 chosen genes and the UMAP was plotted using ggplot2 [@ggplot2]. The colors represented the corresponding tumor type of the patient. We chose to continue with the chosen genes.

A first rough model was trained using all 20 of the chosen genes and the glm [@R-base] function and specifying the model to use a binomial error distribution and a logit link. This model was then passed into the blr_step_aic_both function from the blorr package [@blorr]. This function calculates the best composition of the given 20 genes. With the best configuration the final model was trained on the training dataset.

To evaluate the model the first step was to predict whether the patients of the testing dataset were LUAD patients. For this the native predict function [@R-base] was used. The resulting probabilities were transformed into predictions for 1 or 0 using a cutoff value of 50%. Next a confusion table was used to estimate the false-positive and false-negative rates using the known tumor type of each patient and comparing that to the prediciton of the model. As a final evaluation step the package ROCR [@rocr] was used to create a ROC curve clearly showing the performance of the model.

<!-- add your chapters here -->

\tableofcontents

# Results

## Cancer hallmark pathways

By calculating the Jaccard index for each metabolism gene set to each hallmark gene set, the similarity between these pathways was measured and visualized in a heatmap (**Fig. XXX**). This highlights that there is a general low similarity between the selected pathways and only a few gene sets show a slightly higher similarity which don't exceed an index of 0.2. The most shared genes are found in the alanine, aspartate and glutamate metabolism with glutamine metabolism. Additionally, large overlaps are found in purine and pyrimidine metabolism with genome repair and down regulation as well as in lipid and fatty acid metabolism with VEGF-induced angiogenesis.

```{r test, echo = FALSE, message=FALSE, fig.align='center', fig.cap='\\textbf{Jaccard Coefficients of hallmark gene sets and metabolism gene sets.} The x-axis is defined by the given hallmark gene sets, whereas the y-axis assigned to the selected metabolism heatmap.', out.width='0.75\\linewidth'}

knitr::include_graphics(".\output\2_geneset_heatmap_metabolism_vs_hallmarks.pdf")
```

# Discussion

# Outlook

## Analysis of metastasis formation in LUAD

About a third of LUAD patients already present with brain metastases at the time of diagnosis and about half of all patients will eventually develop brain metastases [@shih2020]. As a cancer's ability to metastasize dramatically impacts a patient's chances of survival, a comprehensive analysis of the genetic alterations that most often lead to metastasis formation would allow to single out patients at risk for developing metastases and to treat them accordingly.

The comparison of genomic expression profiles of cancer cells taken from the metastases with those taken from the primary tumor might reveal which mutations enable a lung cancer cell to metastasize.

## Smoker vs non-smoker

It is a well-known fact that lung cancer is a smoker's disease. However, in recent years studies have found that lung cancer incidence is decreasing in smokers and increasing in non-smokers. Furthermore, the same study states that the genomic profile of lung cancer in non-smokers differs from that in smokers. [@qiu2015]. Inspired by this study, a possible next step would be to subgroup the data into smokers and non-smokers and to compare the two groups in order to determine which pathways are differentially expressed and if the results of Qui *et al* can be replicated with our data.

## Identification of Immune Subtypes

Many different research groups have already set out to subtype LUAD according to immune signature defined for instance by PD-L1 expression or immune cell infiltration. Generally, the more pronounced a cancer's immune signature is, the better the cancer will respond to immunotherapy and the better a patient's chances of survival [@xu2020]. Based on the findings of the likes of Xu *et al.*, our own data could be divided according to immune signature and determining trends within each subgroup such as survival rate could be identified.

## Find defining trends between LUAD clusters

Performance of UMAP and PCA on our data (**Figure XXX**) showed that LUAD forms two distinct clusters. Further analysis may reveal the genetic differences within LUAD that lead to clustering as well as the genetiv similarities shared by samples belonging to the same cluster. Additionally, response to therapy might differ between clusters and thus patients belonging to one cluster or another might face vastly different chances of survival.

## Prediction of cancer stage

Over the course of this project we trained a logistic regression model to predict whether an individual is at risk of eventually developing LUAD. This model could be further sophisticated to additionally predict the cancer stage on the grounds of a patient's genomic profile as well as other factors like age or smoking habits. With enough training, this model could ideally be used as a less invasive alternative to the current diagnostic methods and therefore help determine an adequate treatment plan with reduced patient trauma.

## Epigenetics

The term epigenetics describes hereditary changes in gene expression that are not due to changes in the DNA sequence. The most common epigenetic alterations in cancer cells include global hypomethylation of repetitive DNA sequence regions and hypermethylation of tumor suppressor genes which are consequently inactivated [@esteller2008]. Analysis of the differences in methylation patterns between tumorous and healthy tissue would allow to determine which tumor suppressor genes are most commonly inactivated in LUAD. Furthermore differences in methylation patterns between LUAD and other cancer types could further help to distinguish the process of LUAD development from that of other cancers.

## Experimental validation

The ultimate step of any analysis would be to seek empirical confirmation of novel findings. Since TCGA holds immunohistochemistry stained samples of tumor tissue it would be possible to directly study in what way different genomic profiles impact the development of tumors *in vivo*.

# References

::: {#refs}
:::

# Appendix
