---
title: " "
output:
  pdf_document:
    number_sections: TRUE
  html_document:
    code_folding: hide
documentclass: scrreprt
classoption: [parskip, oneside] # für drucken twoside
header-includes:
  - \usepackage[ngerman, main=english]{babel}
  - \usepackage[utf8]{inputenc}
  - \usepackage[T1]{fontenc}
  - \usepackage{lmodern}
  - \usepackage[onehalfspacing]{setspace}
  - \usepackage[left=2.50cm, right=2.50cm, top=2.50cm, bottom=2.50cm, bindingoffset=10mm, includehead, includefoot]{geometry}
  - \usepackage[headsepline]{scrlayer-scrpage}
  - \usepackage{url}
  - \usepackage[backend=biber, style=authoryear, giveninits=true, maxbibnames=99, uniquename=init, maxcitenames=2, hyperref=true, date=year]{biblatex}
  #- \usepackage{bib/cellstyle_it}
  - \usepackage{setspace}
  - \usepackage{xpatch}
  - \usepackage{csquotes}
  - \usepackage{amsmath}
  - \usepackage{listings}
  - \usepackage{booktabs}
  - \usepackage{longtable}
  - \usepackage{multirow}
  - \usepackage{rotating}
  - \usepackage{subfigure}
  - \usepackage{graphicx}
  - \usepackage{float}
  - \usepackage{acronym}
  - \usepackage{lipsum}
  - \usepackage{scrhack}
  - \emergencystretch=50pt
  - \clubpenalty = 10000
  - \widowpenalty = 10000
  - \displaywidowpenalty = 10000
  - \automark[section]{chapter}
  - \renewcommand*{\chaptermarkformat}{}
  - \renewcommand*{\sectionmarkformat}{}
  - \setkomafont{title}{\sffamily}
  - \setkomafont{disposition}{\usekomafont{title}}
  - \setkomafont{author}{\usekomafont{title}}
  - \setkomafont{date}{\usekomafont{title}}
  - \setkomafont{caption}{\sffamily\small}
  - \setkomafont{captionlabel}{\usekomafont{caption}\bfseries\small}
  - \setkomafont{pagehead}{\normalfont\scshape}
csl: [report/cell.csl]
bibliography: [report/packages.bib, report/Citations.bib]
---

```{=tex}
\begin{titlepage}
\centering
    {\Large Ruprecht-Karls-Universität Heidelberg\\
        Fakultät für Biowissenschaften\\
        Bachelorstudiengang Molekulare Biotechnologie\\}

    {\vspace{\stretch{2}}}
    {\doublelespacing{\usekomafont{title}}

        {\Huge Cancer Hallmark and Metabolic Pathways in Cancer}

      {\Huge Topic 02 Team 03}

        {\Huge Exploration of Lung Adenocarcinoma (LUAD)}

    }

    \vspace{\stretch{2}}
    {\Large Data Science Project SoSe 2022}

    \vspace{\stretch{2}}

    {\Large
        \begin{tabular}{rl}
            Autoren & Paul Brunner, Marie Kleinert, Felipe Stünkel, Chloé Weiler\\
            Abgabetermin &18.07.2022\\
        \end{tabular}
    }

    \vspace{\stretch{1}}

\end{titlepage}
```
# Introduction

## Biological Background

## Principal Component Analysis (PCA)

Principal component analysis (PCA) is a procedure used to perform linear dimension reduction. The goal is to reduce the dimension of a given data set whilst losing as little information as possible by retaining a maximum of the standardized data set's variation [@ringner2008].

Principal components (PC) are a set of new orthogonal variables that are made up of a linear combination of the original variables. Principal components display the pattern of similarity of the observations and of the variables as points in maps [@abdi2010]. By convention, the PCs are ordered in decreasing  order according to the amount of variation they explain of the original data [@ringner2008]. It is important to note that all PCs are uncorrelated.

PCA is a useful tool for genome-wide expression studies and often serves as a first step before clustering or classification of the data. Dimension reduction is a necessary step for easy data exploration and visualization [@ringner2008].

## Uniform Manifold Approximation and Projection (UMAP)

Uniform manifold approximation and projection (UMAP) is a k-neighbour graph based algorithm that is used for nonlinear dimension reduction [@smets2019] [@mcinnes2018].

After data normalization, the Euclidean distances between points in a two-dimensional space of the graph are calculated and a local radius is determined [@VERMEULEN2021119547]. In general the closer two points are to each other, the more similar they are. UMAP makes a density estimation to find the right local radius. This variable radius is smaller in high density regions of data points and larger in low density regions. In general, the density is higher when the k-nearest neighbour is close and vice versa. The number of k-nearest neighbours controls the number of neighbours whose local topology is preserved. Precisely, a large number of neighbours will ensure that more global structure is preserved whereas a smaller number of neighbours will ensure the preservation of more local structure [@mcinnes2018].

UMAP is a newer method tan PCA and it is generally believed to be easier to interpret and group data than by using PCA. Furthmore, UMAP has the advantage of not requiring linear data [@milosevic2022].

## Gene Set Enrichment Analysis (GSEA)

Gene set enrichment analysis (GSEA) is a computational method that is used to determine whether two gene expression states are significantly different from each other or not. In this project we compared gene expression profiles between healthy and tumorous tissue of LUAD[@subramanian2007].

Two data sets are compared and the genes are sorted from the most to the least differential expression between the data sets according to their p-values. This creates a ranked list L.

Referring to an *a priori* defined set of gene sets S, the goal is to locate for each pathway of S where its corresponding genes fall in L and find a discerning trend. If the genes of a given pathway are randomly distributed in L then the pathway is assumed to not significantly contribute to the particular tumor's phenotype. However if the genes are primarily clustered at the top or the bottom of L then a phenotypic significance of the given pathway can be assumed.

To determine the location of the genes, an enrichment score is calculated for each pathway. For this, a running-sum statistic is calculated as the list L is ran through. The running-sum is increased every time a gene belonging to the pathway in question is encountered and decreased otherwise. An enrichment score is thus calculated for each pathway. The enrichment score is defined as the maximum deviation from zero of the running-sum.

Lastly, adjustment for multiple hypothesis testing is performed by normalizing the enrichment score for each pathway to account for its size and a normalized enrichment score is obtained.

GSEA is a useful tool for interpretation of gene expression data.

[@subramanian2005]

## Gene Set Variation Analysis (GSVA)

Gene set variation analysis (GSVA) is an unsupervised method to estimate pathway activities based on gene expression data. Contrarily to the aforementioned GSEA, GSVA does not rely on phenotypic characterisation of the data sets into two categories but rather quantifies enrichment in a sample-wise manner which makes GSVA the better choice to perform on the tcga_exp data set.

GSVA estimates a cumulative distribution for each gene over all samples. The gene expression values are then converted according to these estimated cumulative distributions into scaled values. Based on these new values, the genes are ranked in each sample. Next, the genes are classified into two distributions and a Komogorow-Smirnow statistic is calculated to judge how similar the distributions are to each other and to obtain an ES.

The GSVA corresponds to either the maximum deviation between both running sums or the GSVA score can be defined as the difference of the maximum deviations in the positive and in the negative direction. A highly positive or negative GSVA score indicates that the studied gene set is positively or negatively enriched compared to the genes not in the gene set, respectively. If the GSVA score for a given gene set is close to zero, then the gene set is probably not differentially expressed compared to the genes not in the gene set.

[@hanzelmann2013]

# Abbreviations

|      |                                               |
|------|-----------------------------------------------|
| GSEA | gene set enrichment analysis                  |
| GSVA | gene set variation analysis                   |
| LUAD | lung adenocarcinoma                           |
| PC   | principal component                           |
| PCA  | principal component analysis                  |
| TCGA | The cancer genome atlas                       |
| UMAP | uniform manifold approximation and projection |

# Methods

## Overview of used packages

+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| Package Name    | Application                                                                                | **Reference**      |
+=================+============================================================================================+====================+
| babyplots       | create interactive 3D visualizations                                                       | @babyplots         |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| base            | basic R functions                                                                          | @R-base            |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| bayesbio        | calculate Jaccard coefficients                                                             | @bayesbio          |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| BiocParallel    | novel implementations of functions for parallel evaluation                                 | [@biocparallel]    |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| biomaRt         | access to genome databases                                                                 | @biomaRt2009       |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| cinaR           | combination of different packages                                                          | @cinaR             |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| cluster         | cluster analysis of data                                                                   | @cluster           |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| ComplexHeatmap  | arrange multiple heatmaps                                                                  | [@complexheatmap]  |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| edgeR           | assess differential expression in gene expression profiles                                 | @edgeR             |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| EnhancedVolcano | produce improved volcano plots                                                             | [@enhancedvolcano] |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| enrichplot      | visualization of gene set enrichment results (GSEA)                                        | @enrichplot        |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| FactoMineR      | perform principal component analysis (PCA)                                                 | @FactoMineR        |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| fgsea           | Run GSEA on a pre-ranked list                                                              | @fgsea             |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| ggplot2         | visualization of results in dot plots, bar plots and box plots                             | @ggplot2           |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| ggpubr          | formatting of ggplot2-based graphs                                                         | @ggpubr            |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| grid            | implements the primitive graphical functions that underlie the ggplot2 plotting system     | @grid              |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| gridExtra       | arrange multiple plots on a page                                                           | @gridExtra         |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| GSVA            | Run GSVA on a data set                                                                     | @GSVA              |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| gplots          | plotting data                                                                              | [@gplots]          |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| gtools          | calculate foldchange, find NAs,                                                            | @R-gtools          |
|                 |                                                                                            |                    |
|                 | logratio2foldchange                                                                        |                    |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| knitr           | creation of citations using write_bib                                                      | @knitr2014         |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| limma           | "linear models for microarray data"                                                        | @limma2015         |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| msigdbr         | provides the 'Molecular Signatures Database' (MSigDB) gene sets                            | @msigdbr           |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| parallel        | allows for parallel computation through multi core processing                              | @parallel          |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| pheatmap        | draw clustered heatmaps                                                                    | @pheatmap          |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| RColorBrewer    | provides color schemes for maps                                                            | @RColorBrewer      |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| scales          | helps in visualization: r automatically determines breaks and labels for axes and legends  | @scales            |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| Seurat          | visualize gene set enrichment results in dot plots                                         | @R-SeuratObject    |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| tidyverse       | collection of R packages, including ggplot2                                                | @tidyverse         |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+
| uwot            | performs dimensionality reduction and Uniform Manifold Approxiamtion and Projection (UMAP) | @uwot              |
+-----------------+--------------------------------------------------------------------------------------------+--------------------+

: **Tab. 1: Used packages in alphabetical order.**

## Our Data

<!-- add your chapters here -->

\tableofcontents

# Result

# Discussion

# Outlook

## Analysis of metastasis formation in LUAD

About a third of LUAD patients already present with brain metastases at the time of diagnosis and about half of all patients will eventually develop brain metastases \[@shih2020\]. As a cancer's ability to metastasize dramatically impacts a patient's chances of survival, an comprehensive analysis of the genetic alterations that most often lead to metastasis formation would allow to single out patients at risk of developing metastases early on.

The comparison of genomic expression profiles of cancer cells taken from the metastases with those of the primary tumor might reveal which mutations enable a lung cancer cell to metastasize.

## Smoker vs non-smoker

It is a well-known fact that lung cancer is a smoker's disease. However, in recent years studies have found that lung cancer incidence is decreasing in smokers and increasing in non-smokers. Furthermore, the same study states that the genomic profile of lung cancer in non-smokers differs from that in smokers. \[@qiu2015\]. Inspired by this study, a possible approach would be to subgroup our data into smokers and non-smokers and to compare the two groups in order to determine which pathways are differentially expressed and if the results of Qui *et al* can be replicated with our data.

## Immune signature

Many different research groups have set out to subtype LUAD according to immune signature

## Survival and therapy

## Prediction of cancer stage

Over the course of this project we trained a logistic regression model to predict whether an individual is at risk of eventually developing LUAD. This model could be further sophisticated to additionally predict the cancer stage on the grounds of a patient's genomic profile as well as other factors like age or smoking habits. With enough training, this model could ideally be used as a less invasive alternative to the current methods in LUAD diagnosis and therefore help determine an adequate treatment plan with less patient trauma.

## Epigenetics

The term epigenetics describes hereditary changes in gene expression that are not due to changes in the DNA sequence. The most common epigenetic alterations in cancer cells include global hypomethylation of repetitive DNA sequence regions or hypermethylation of tumor suppressor genes which are thus inactivated \[@esteller2008\]. Analysis of differences in methylation pattern between tumorous and healthy tissue would allow to determine which tumor suppressor genes are most commonly inactivated in LUAD. Furthermore differences in methylation patterns between LUAD and other cancer types could further help to distinguish the process of LUAD development from that of other cancers.

## Experimental validation

The ultimate step of any analysis would be to seek empirical confirmation of novel findings. Since TCGA holds immunohistochemistry stained samples of tumor tissue it would be possible to directly study in what way different genomic profiles impact the development of tumors *in vivo*.

# References

::: {#refs}
:::

# Appendix
